### PR TITLE
Pipenv to poetry

### PR DESCRIPTION
**Pull request by y0urself**
The template is missing here ...

Added poetry support, removed pipenv support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ Python Package Index using [pip](https://pip.pypa.io/):
 Note the `pip` refers to the Python 3 package manager. In a environment where
 Python 2 is also available the correct command may be `pip3`.
 
-### Install using pipenv
+### Install using poetry
 
-It is highly encouraged to use [pipenv](https://github.com/pypa/pipenv) for
-maintaining your project's dependencies. Normally autohooks-plugin-pylint is
+It is highly encouraged to use [poetry](https://python-poetry.org) for
+maintaining your project's dependencies. Normally autohooks-plugin-black is
```
Comment (bjoernricks) [inline README.md:25]:
```suggestion
maintaining your project's dependencies. Normally autohooks-plugin-pylint is
```

**Review by bjoernricks — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Python 2 is also available the correct command may be `pip3`.
 ### Install using poetry
 
 It is highly encouraged to use [poetry](https://python-poetry.org) for
-maintaining your project's dependencies. Normally autohooks-plugin-black is
+maintaining your project's dependencies. Normally autohooks-plugin-isort is
```
Comment (bjoernricks) [inline README.md:25]:
```suggestion
maintaining your project's dependencies. Normally autohooks-plugin-pylint is
```

**Review by bjoernricks — CHANGES_REQUESTED** [review]
Still the wrong name ;-)

File: README.md (hunk 2 above)
Comment (y0urself) [reply README.md:25]: :(

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**